universal-application-tool-0.0.1/app/services/applicant/question/RepeaterQuestion.java

```diff
@@ -39,13 +38,11 @@ public RepeaterQuestionDefinition getQuestionDefinition() {
     return (RepeaterQuestionDefinition) applicantQuestion.getQuestionDefinition();
   }
 
-  public Path getRepeaterPath() {
-    return getQuestionDefinition().getPath();
-  }
-
   @Override
+  /** This is answered if there is at least one entity name stored. */
```

Comment:
nit: move the docstring above the `@Override`

Reply:
oops!